build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+npm i;
```

Comment:
shell scripts don't need semicolons unless you want to put more than one command in a line

Reply:
Oh.. I knew you would complain..

Reply:
Yeah, I am aware of it but I always prefer to use them ;p